pom.xml

```diff
@@ -572,5 +572,97 @@
+                <goals>
+                  <goal>aggregate</goal>
+                </goals>
+              </execution>
+            </executions>
+            <configuration>
+              <docletArtifact>
+                <groupId>com.google.doclava</groupId>
+                <artifactId>doclava</artifactId>
+                <version>1.0.6</version>
+              </docletArtifact>
+              <doclet>com.google.doclava.Doclava</doclet>
+              <bootclasspath>${sun.boot.class.path}</bootclasspath>
+              <additionalDependencies>
+                <additionalDependency>
+                  <groupId>com.google.j2objc</groupId>
+                  <artifactId>j2objc-annotations</artifactId>
+                  <version>1.3</version>
+                </additionalDependency>
+              </additionalDependencies>
+              <additionalOptions>
+                <additionalOption>-hdf</additionalOption>
+                <additionalOption>book.path</additionalOption>
+                <additionalOption>/java/_book.yaml</additionalOption>
+                <additionalOption>-hdf</additionalOption>
+                <additionalOption>project.path</additionalOption>
+                <additionalOption>/_project.yaml</additionalOption>
+                <additionalOption>-hdf</additionalOption>
+                <additionalOption>devsite.path</additionalOption>
+                <additionalOption>/java/docs/reference/</additionalOption>
+                <additionalOption>-d</additionalOption>
+                <additionalOption>${project.build.directory}/apidocs</additionalOption>
+                <additionalOption>-templatedir</additionalOption>
+                <additionalOption>${devsite.template}</additionalOption>
+                <additionalOption>-toroot</additionalOption>
+                <additionalOption>/java/docs/reference/</additionalOption>
+                <additionalOption>-yaml</additionalOption>
+                <additionalOption>_toc.yaml</additionalOption>
+                <additionalOption>-warning</additionalOption>
+                <additionalOption>101</additionalOption>
+              </additionalOptions>
+              <useStandardDocletOptions>false</useStandardDocletOptions>
+              <additionalJOption>-J-Xmx1024m</additionalJOption>
+            </configuration>
+          </plugin>
+          <plugin>
+            <!-- Clean up some references and files. -->
+            <artifactId>maven-antrun-plugin</artifactId>
+            <version>1.7</version>
+            <executions>
+              <execution>
+                <phase>site</phase>
+                <goals>
+                  <goal>run</goal>
+                </goals>
+                <configuration>
+                  <target>
+                    <echo message="Updating relative links in API docs" />
+                    <replace dir="${project.build.directory}/apidocs" token="href=&quot;com" value="href=&quot;/java/docs/reference/com" />
```

Comment:
This repo config is responsible for artifacts that also publish classes under `io` and `google` (for proto files). We should also account for those packages being re-rooted along with `com`.